ONSdigital/census-rm-pubsub
# Basic unit tests

## Motivation and Context
This contains a few unit tests as a starter

## What has changed
Rabbit_helper and subscriber unit tests

## How to test?
clone project, checkout this branch, cd into project
pipenv install
pipenv shell
pytest test/

This should produce 3 passed & 1 warnings. #jcox is happy with the warning at the moment as it's in a package we're using not our code
GCP have got ahead of them for now
